### PR TITLE
Clean up formatting

### PR DESCRIPTION
**Pull request by jeffbyrnes**
@complexlogic this is entirely optional, but while I was poking around, did some cleanup per my linters, as well as the auto-formatting that comes with the Visual Studio Code C++ extension.

Feel free to take it or leave it, no hurt feelings if you don’t like these & prefer not to.

### CODE REVIEW

**Comment by complexlogic** [comment]
@jeffbyrnes The Windows build is failing. The reason is because the filename of the CHANGELOG was changed, but the CMake script was not updated accordingly. The CHANGELOG ships in the Windows .zip packages. For that reason, I prefer to leave the CHANGELOG file as plain text and not convert it to Markdown because not everybody will view it in their web browser on GitHub.

Regarding https://github.com/complexlogic/rsgain/pull/36/commits/277092cee6fb142de5f4a5c0569a702f84a0a58d, there is no objectively correct coding style in C or C++; it's a matter of personal preference. I prefer 4 space indentations because I think it makes the code more readable. Whoever wrote that VSCode extension is entitled to their preferences also, but I would like to keep my existing coding style.

Other than the two issues noted above, these changes look fine. I can merge this if you revert those changes.

**Comment by jeffbyrnes** [comment]
@complexlogic alrighty, dropped the code formatting changes (I figured those wouldn’t fly but I was curious) and reverted the `CHANGELOG` to Markdown bits.

File: docs/BUILDING.md [hunk 1]
```diff
@@ -1,98 +1,131 @@
...
+```bash
 sudo apt install libebur128-dev libtag1-dev libavformat-dev libavcodec-dev libswresample-dev libavutil-dev libfmt-dev libinih-dev
 ```
+
 ### Pacman-based Linux (Arch, Manjaro, etc.)
-```
+
+```bash
 sudo pacman -S libebur128 taglib ffmpeg fmt libinih
-```
```
Comment (complexlogic) [inline docs/BUILDING.md:30]: You need to close this code block.

**Review by complexlogic — CHANGES_REQUESTED** [review]

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,5 @@
 v3.1 (2022-11-4)
+
```
Comment (complexlogic) [inline CHANGELOG]: Since this is no longer markdown, I prefer to not have the spaces between the release and the changes. Please revert this and the rest.

**Review by complexlogic — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 2 above)
Comment (jeffbyrnes) [reply CHANGELOG]: You got it, sorry for missing that!

**Comment by jeffbyrnes** [comment]
Further tweaked per your comments.

**Pull request merged**